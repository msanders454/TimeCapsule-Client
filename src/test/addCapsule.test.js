/* eslint-disable no-undef */
import React from "react";
import { shallow, configure } from "enzyme";
import AddCapsulePage from '../Components/addCapsule/addCapsule';
import Adapter from "enzyme-adapter-react-16";
 
configure({ adapter: new Adapter() });
test("should render AddCapsule correctly", () => {
  const wrapper = shallow(<AddCapsulePage />);
  expect(wrapper).toMatchSnapshot();
});