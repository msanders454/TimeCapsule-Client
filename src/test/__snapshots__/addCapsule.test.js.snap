// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddCapsule correctly 1`] = `ShallowWrapper {}`;
